### DISCUSSION TITLE
Updates for Test Console "Try It" section

### DISCUSSION THREAD

**Discussion by ng-da** [Azure/api-management-developer-portal#2631] [Feature request]
Hi could you pls update/make configurable this templates, so we will be able to add our own examples/new version of Python, etc. ?

https://github.com/Azure/api-management-developer-portal/tree/c6d71e3a6f5a572bd383888b3a099bf4f44c7812/src/components/operations/operation-details/ko/runtime/templates

**Comment by sthirthala** [comment]
Thanks. We will look at this as a feature request.
